### PR TITLE
Dev/iwu/us vbs22 37 cambio y recuperacion de contraseña

### PR DESCRIPTION
**Pull request by ichivancrCenfotec**
Múdulos de cmabio y recuperación de contraseña. Front End

### CODE REVIEW

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -63,6 +77,16 @@ export const routes: Routes = [
           ],
           name: 'Dashboard'
         }
+      },
+      {
+        path: 'update-user',
+        component: UpdateUserComponent,
+        data: { 
+          authorities: [
```
Comment (nefi-cenfo) [inline src/app/app.routes.ts:85]: you can remove the authorities attribute if is not needed.

File: src/app/components/user/user-from/user-form.component.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h3 class="mb-5 px-5">{{title}}</h3>
         <input #lastname="ngModel" type="text" class="form-control" id="lastname" 
           name="lastname" 
           placeholder="Hernández"
-          [(ngModel)]="user.lastname" 
+          [(ngModel)]="user.last_name"
```
Comment (nefi-cenfo) [inline src/app/components/user/user-from/user-form.component.html:52]: Please use camel case to variable names.

File: src/app/interfaces/index.ts [hunk 3]
```diff
@@ -8,15 +8,33 @@ export interface IResponse<T> {
...
+  operational?: boolean;
+  creation_date?: string;
+  creation_resposible?: number;
+  last_update_datetime?: string;
+  upodate_resposible?: number;
   authorities?: IAuthority[];
+  otp?: string;
+
+ 
+  // id?: number;
```
Comment (nefi-cenfo) [inline src/app/interfaces/index.ts:28]: Is this commented code needed?

File: src/app/pages/auth/login/login.component.html [hunk 4]
```diff
@@ -35,32 +32,45 @@ <h4>Sign in to account</h4>
...
           </form>
         </div>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+  <footer class="footer">
+    <div class="footer-line"></div>
+    <!-- <img src="../../../../assets/icons/Round_Voiage.webp" alt="Logo" class="footer-logo"> -->
```
Comment (nefi-cenfo) [inline src/app/pages/auth/login/login.component.html:73]: Is this commented code needed?

File: src/app/pages/auth/login/login.component.scss [hunk 5]
```diff
@@ -44,4 +72,100 @@
...
+/* Footer */
+.footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  // background: transparent;
```
Comment (nefi-cenfo) [inline src/app/pages/auth/login/login.component.scss:145]: Is this commented code needed?

File: src/app/pages/auth/login/login.component.scss [hunk 6]
```diff
@@ -44,4 +72,100 @@
...
+  // background: #775D68;
+  background: #FADCD4;
+  background-size: cover;
+  background-position: center;
+  margin-left: 0;
+  // flex-direction: column;
+}
+
+// .footer-line {
```
Comment (nefi-cenfo) [inline src/app/pages/auth/login/login.component.scss:154]: Is this commented code needed?

File: src/app/pages/resetpassword/resetpassword.component.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+              </button>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <footer class="footer">
+    <div class="footer-line"></div>
+    <!-- <img src="../../../../assets/icons/Round_Voiage.webp" alt="Logo" class="footer-logo"> -->
```
Comment (nefi-cenfo) [inline src/app/pages/resetpassword/resetpassword.component.html:46]: Is this commented code needed?

File: src/app/pages/resetpassword/resetpassword.component.scss [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+/* Footer */
+.footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  // background: transparent;
```
Comment (nefi-cenfo) [inline src/app/pages/resetpassword/resetpassword.component.scss:145]: Is this commented code needed?

File: src/app/pages/resetpassword/resetpassword.component.scss [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+  // background: #775D68;
+  background: #FADCD4;
+  background-size: cover;
+  background-position: center;
+  margin-left: 0;
+  // flex-direction: column;
+}
+
+// .footer-line {
```
Comment (nefi-cenfo) [inline src/app/pages/resetpassword/resetpassword.component.scss:154]: Is this commented code needed?

File: src/app/pages/resetpassword/resetpassword.component.scss [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+//   height: 5px;
+//   background: linear-gradient(to left, #FADCD4 0%, transparent 100%);
+//   width: 100%;
+// }
+
+.footer-logo {
+  width: 2.5vw; 
+  height: 5vh;
+
+  // margin-top: 5px;
```
Comment (nefi-cenfo) [inline src/app/pages/resetpassword/resetpassword.component.scss:165]: Is this commented code needed?

File: src/app/pages/update-user/update-user.component.html [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+                </button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+    <footer class="footer">
+      <div class="footer-line"></div>
+      <!-- <img src="../../../../assets/icons/Round_Voiage.webp" alt="Logo" class="footer-logo"> -->
```
Comment (nefi-cenfo) [inline src/app/pages/update-user/update-user.component.html:75]: Is this commented code needed?

File: src/app/pages/update-user/update-user.component.ts [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+public userError!: string;
+
+@Output () user: IUser = {};
+
+constructor(
+  private router: Router
+) {}
+
+public updateUserForm : {name: string; last_name:string; second_last_name:string;} = {
```
Comment (nefi-cenfo) [inline src/app/pages/update-user/update-user.component.ts:35]: Please use an Interface as object type.

File: src/app/pages/validate-otp/validate-otp.component.html [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+                </button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+    <footer class="footer">
+      <div class="footer-line"></div>
+      <!-- <img src="../../../../assets/icons/Round_Voiage.webp" alt="Logo" class="footer-logo"> -->
```
Comment (nefi-cenfo) [inline src/app/pages/validate-otp/validate-otp.component.html:60]: Is this commented code needed?

**Review by nefi-cenfo — DISMISSED** [review]

File: src/app/interfaces/index.ts (hunk 3 above)
Comment (ichivancrCenfotec) [reply src/app/interfaces/index.ts:28]: No

File: src/app/pages/validate-otp/validate-otp.component.html (hunk 13 above)
Comment (ichivancrCenfotec) [reply src/app/pages/validate-otp/validate-otp.component.html:60]: No

File: src/app/pages/auth/login/login.component.scss (hunk 5 above)
Comment (ichivancrCenfotec) [reply src/app/pages/auth/login/login.component.scss:145]: No

File: src/app/pages/auth/login/login.component.scss (hunk 6 above)
Comment (ichivancrCenfotec) [reply src/app/pages/auth/login/login.component.scss:154]: No

File: src/app/pages/resetpassword/resetpassword.component.html (hunk 7 above)
Comment (ichivancrCenfotec) [reply src/app/pages/resetpassword/resetpassword.component.html:46]: No

File: src/app/pages/update-user/update-user.component.html (hunk 11 above)
Comment (ichivancrCenfotec) [reply src/app/pages/update-user/update-user.component.html:75]: No

File: src/app/pages/auth/login/login.component.html (hunk 4 above)
Comment (ichivancrCenfotec) [reply src/app/pages/auth/login/login.component.html:73]: No

File: src/app/components/user/user-from/user-form.component.html (hunk 2 above)
Comment (ichivancrCenfotec) [reply src/app/components/user/user-from/user-form.component.html:52]: Named as backend variable

**Pull request closed without merging**